src/Driver/driver_configs.jl

```diff
@@ -351,16 +361,21 @@ function SingleStackConfiguration(
     numerical_flux_first_order = RusanovNumericalFlux(),
     numerical_flux_second_order = CentralNumericalFluxSecondOrder(),
     numerical_flux_gradient = CentralNumericalFluxGradient(),
+    grid_stretching = SVector{3}(
+        NoStretching(),
+        NoStretching(),
+        NoStretching(),
+    ),
 ) where {FT <: AbstractFloat}
 
     print_model_info(model)
 
     xmin, xmax = zero(FT), one(FT)
     ymin, ymax = zero(FT), one(FT)
     brickrange = (
-        grid1d(xmin, xmax, nelem = 1),
-        grid1d(ymin, ymax, nelem = 1),
-        grid1d(zmin, zmax, nelem = nelem_vert),
+        grid1d(xmin, xmax, grid_stretching[1], nelem = 1),
+        grid1d(ymin, ymax, grid_stretching[1], nelem = 1),
+        grid1d(zmin, zmax, grid_stretching[1], nelem = nelem_vert),
```

Comment:
Did you intend to use `grid_stretching[1]` for all of these?

Reply:
No, I'll fix that shortly